aneris/utils.py

```diff
@@ -439,14 +441,15 @@ def _add_totals(self, totals):
         grp_idx = [x for x in df_idx if x != 'sector']
         rows = self.df.groupby(grp_idx).sum().reset_index()
         rows['sector'] = totals
-        self.df = self.df.append(rows)
+        self.df = pd.concat([self.df, rows])
 
     def _add_aggregates(self):
         mapping = pd_read(iamc_path('sector_mapping.xlsx'),
                           sheet_name='Aggregates')
         mapping = mapping.applymap(remove_emissions_prefix)
 
         rows = pd.DataFrame(columns=self.df.columns)
+        rows = []
```

Comment:
```suggestion
        rows = []
```

Ups, forgot to remove a line.